qgis_hub_plugin/gui/resource_browser.py

```diff
@@ -173,32 +178,36 @@ def show_preview(self):
         self.groupBoxPreview.show()
 
     def download_resource(self):
+        # pydevd_pycharm.settrace('127.0.0.1', port=53100, stdoutToServer=True, stderrToServer=True)
```

Comment:
I know it's for remote debugging, but we shouldn't add this kind of comment/code in the repository.

Reply:
I had for got to remove it.
Resolved in the new commit.